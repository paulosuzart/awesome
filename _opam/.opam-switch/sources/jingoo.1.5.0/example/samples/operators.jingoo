1 + 1 = {{ 1 + 1 }}
1 - 1 = {{ 1 - 1 }}
2 * 3 = {{ 2 * 3 }}
5 / 2 = {{ 5 / 2 }}
2 ** 8 = {{ 2 ** 8 }}
8 % 3 = {{ 8 % 3 }}

"a" + "b" = {{ "a" + "b" }}
{% for i in [1,2] + [3,4] %}{{i}}{% endfor %}
({a:"foo"} + {b:"bar"}).a = {{ ({a:"foo"} + {b:"bar"}).a }}
({a:"foo"} + {b:"bar"}).b = {{ ({a:"foo"} + {b:"bar"}).b }}
({a:"foo"} + {a:"bar"}).a = {{ ({a:"foo"} + {a:"bar"}).a }}
