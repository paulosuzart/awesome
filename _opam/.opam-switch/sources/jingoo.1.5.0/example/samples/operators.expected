1 + 1 = 2
1 - 1 = 0
2 * 3 = 6
5 / 2 = 2
2 ** 8 = 256.
8 % 3 = 2

"a" + "b" = ab
1234
({a:"foo"} + {b:"bar"}).a = foo
({a:"foo"} + {b:"bar"}).b = bar
({a:"foo"} + {a:"bar"}).a = bar
